src/pages/blog/starlingx-release-4.md

```diff
@@ -0,0 +1,35 @@
+
+Security is also crucial when you manage your hardware infrastructure for instance when you boot or restart the servers that are part of your deployment. Redfish provides well-known standards in this area to provide solutions to this challenge. StarlingX now supports the Redfish Virtual Media Controller that provides you with the option to use a secure BMC-based ISO image boot process.
+
+Automation is highly important in edge computing use cases due to a lot of locations being remote with limited possibility for maintenance. StarlingX now provides a [certification manager](https://docs.starlingx.io/configuration/cert_config.html) that enables automated certification issuance as well as monitoring the expiration dates and giving the possibility to auto-renew.
+
+To provide more flexibility for users the community is working on providing support for external systems in areas where tools exist already. One example to this is backends used during authentication and authorization. You can now configure StarlingX to use Windows Active Directory for authentication of the  Kubernetes API.
+
+Kubernetes is also supported now in the [backup and restore](https://docs.starlingx.io/developer_resources/backup_restore.html) functionality of the StarlingX platform, you can now configure it for containerized workloads.
+
+As you may already know StarlingX has a lot of components that you can combine in the configuration that fulfills your needs the best. In the past the build process was monolithic therefore in case of any changes you had to rebuild all components to get the updated image. Now there is support for a [layered build](https://docs.starlingx.io/developer_resources/layered_build_guide.html) where the tool decomposes the system into layers which then can be built independently saving you time as well as resources.
+
+As in all release cycles there are a few components that have new versions available. As part of upgrades you will have a newer Linux kernel in the platform, 4.18 and the latest release of OpenStack called Ussuri is also available.
+
+For the complete list of updates and new features in StarlingX 4.0 check out the [release notes](https://docs.starlingx.io/releasenotes/r4_release.html) and further [documentation](https://docs.starlingx.io) of the project.
```

Comment:
For the complete list of updates and new features in StarlingX R4.0, check out the [release notes](https://docs.starlingx.io/releasenotes/r4_release.html) and the [project documentation](https://docs.starlingx.io).